src/components/SignUp.css

```diff
@@ -0,0 +1,18 @@
+@import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
It might be a good idea for you and Anvinh to figure out how you both want to organize your CSS code. That way, it is more consistent.

Reply:
Yeah we'll definitely figure this out soon.